lib/core/imports/index.js

```diff
@@ -7,6 +7,11 @@ import Color from 'colorjs.io';
 import es6promise from 'es6-promise';
 import { Uint32Array } from 'typedarray';
 import 'weakmap-polyfill';
+import hasOwn from 'core-js-pure/actual/object/has-own';
+
+if (!('hasOwn' in Object) {
```

Comment:
My bad, seems I forgot the end bracket :D

```suggestion
if (!('hasOwn' in Object)) {
```

Reply:
serves me right for naively copypasting